### PR TITLE
Partition monitoring release

### PR DESCRIPTION
**Pull request by thanche1**
Add promtail to release vector.
Change loki helm chart.

### CODE REVIEW

File: release.yaml [hunk 1]
```diff
@@ -173,11 +173,14 @@ docker-images:
...
+        name: grafana/promtail
+        tag: 2.7.3
 helm-charts:
   logging:
-    loki-stack:
-      repository: https://grafana.github.io/helm-charts
-      version: 2.8.9
+    loki:
+      repository: https://grafana.github.io/loki/charts
+      version: 2.8.0
```
Comment (Gerrit91) [inline release.yaml:183]: In https://github.com/metal-stack/metal-roles/pull/150 we use the same chart repository URL and version `4.6.1`. Is this change intended or is it the same?

Comment (robertvolkmann) [reply release.yaml:187]: loki charts are deprecated!

**Review by robertvolkmann — CHANGES_REQUESTED** [review]

**Review by Gerrit91 — APPROVED** [review]
